note-app/src/Reducer/setnoteReducer.js

```diff
@@ -24,14 +24,17 @@ const setnoteReducer = (state,action) => {
                     return {...state,priority: action.payload}
                     case "PRIORTY":
                       return {...state,priority: action.payload}
+                      case "PINTOGGLE":
+                        return {...state,pinned : !state.pinned}
 
                     case "RESET":
                       return {
                    title: '',
                    text: '',
                  noteColor: '',
                    tags: '',
-                 priority: 'High'
+                 priority: 'High',
+                 pinned: false
                 }
          default:
           return state;
```

Comment:
throw an error saying `action type not found` instead of returning state

Reply:
will do that